lib/rocket_chat/request_helper.rb

```diff
@@ -29,6 +29,7 @@ def request_json(path, options = {})
       check_response response, fail_unless_ok
 
       response_json = JSON.parse(response.body)
+      options[:debug].puts("Response: #{response_json.inspect}") if options[:debug]
```

Comment:
Probably safer with `if options[:debug].respond_to? :puts` ?? But then the user has to have some responsibility, right??